components/ComingSoonImage.js

```diff
@@ -2,7 +2,7 @@ import styled, { useTheme } from 'styled-components';
 import MotionLogo from './styles/MotionLogo.styled';
 
 const ImageContainer = styled.div`
-	margin-bottom: ${({ theme }) => theme.spaces.xxl};
+	margin-bottom: var(--space-xxl);
```

Comment:
decomposing was a nice move but this is big brain

Reply:
What I really wanted to do was use it for the media queries but alas, they don't currently support CSS variables. There's a [WC3 standards draft](https://drafts.csswg.org/css-env-1/) in the works that would solve this with environment variables though.